tldr/cli.py

```
@@ -964,6 +974,31 @@ def resolve_language(lang_arg: str, project_path: str | Path) -> str:
+                            lang=sem_lang,
+                            respect_ignore=respect_ignore,
+                        )
+                        print(f"Semantic index: {count} code units")
+                    except ImportError as e:
+                        print(
+                            f"Semantic index skipped (missing dependency: {e}). "
+                            "Install with `pip install llm-tldr[semantic]` or run "
+                            "`tldr warm --no-semantic` to silence this notice.",
+                            file=sys.stderr,
+                        )
+                    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)